controller/script/simulate-proxy/main.go

```diff
@@ -49,10 +49,10 @@ func randomLatencies(count uint32) (latencies []*pb.Latency) {
 	for i := uint32(0); i < count; i++ {
 
 		// The latency value with precision to 100µs.
-		latencyValue := uint32(rand.Int31n(int32(time.Second/(time.Millisecond * 10))))
+		latencyValue := uint32(rand.Int31n(int32(time.Second / (time.Millisecond * 10))))
 		latency := pb.Latency{
 			Latency: latencyValue,
-			Count: rand.Uint32(),
+			Count:   count,
```

Comment:
Good find! Based on the previous implementation of this method (pasted below), I think that we should use `Count: 1` here. We only want to return `count` pb.Latency objects total, each with Count 1.

```
func randomLatencies(count uint32) (latencies []uint32) {
	for i := uint32(0); i < count; i++ {
		latencies = append(latencies, uint32(rand.Int31n(int32(time.Second/time.Microsecond))))
	}
	return
}
```

Reply:
thanks! will do!